doc/source/jupyterhub/installation.md

```diff
@@ -19,7 +19,7 @@ going onwards.
 
 ```{admonition} Introduction to YAML
 If you haven't worked with YAML before, investing some
-minutes [learning about it]([YAML](https://www.youtube.com/watch?v=cdLNKUoMc6c)
+minutes [learning about it]([YAML](https://www.youtube.com/watch?v=cdLNKUoMc6c))
```

Comment:
Hmmm it seems like there is another kind of failure, it is like we have a link label "learning about i" that points to a link with a link label of YAML. Let's remove one label, the YAML label for example.

```suggestion
minutes [learning about it](https://www.youtube.com/watch?v=cdLNKUoMc6c)
```